lib/utils.tsx

```diff
@@ -19,6 +23,30 @@ export const getFilteredThings = (dataset, propertyType) => {
     return ret
 };
 
+/**
+ * checks common name properties on the Thing (VCARD, FOAF) and returns the first it can find
+ * @returns the name of a given Thing, or null if unable to find it
+ */
+export const getThingName = (thing: Thing): string => {
+    const NAME_PROPERTIES = [VCARD.fn, FOAF.name];
+    for(let PROP of NAME_PROPERTIES) {
+        const name = getStringNoLocale(thing, PROP);
+
+        if(name != null) return name;
+    }
+    return null;
+}
+
+/**
+ * @returns a ReactElement, either an img with the FOAF.depcition stored on the Thing, or a placeholder icon
+ */
+export const getThingDepiction = (thing: Thing) : React.ReactElement => {
```

Comment:
This is basically a React component of its own (taking in some inputs/props, spitting out JSX), what do you think to converting it? It'd then be visible in DevTools.